backend/src/controllers/rentalAgency.js

```diff
@@ -0,0 +1,32 @@
+"use strict";
+
+const RentalAgencyModel = require('../models/rentalAgency');
+
+
+const create = (req, res) => {
```

Comment:
We dont need API create method, because we are not goint to have administration for this.

Reply:
Deleted